src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+const listResult = document.querySelector('ul');
+
+const sortList = (list) => {
+  list.sort((a, b) =>
+    (+b.dataset.salary.split(',').join('').replace('$', ''))
+    - (+a.dataset.salary.split(',').join('').replace('$', ''))
+  );
```

Comment:
Не розривай колбек.
І винеси порівнювальні значення в окремі змінні
```suggestion
  list.sort(
    (a, b) => (+b.dataset.salary.split(',').join('').replace('$', ''))
    - (+a.dataset.salary.split(',').join('').replace('$', ''))
  );
```